helloworld-springboot/src/main/java/com/example/java/gettingstarted/HelloworldApplication.java

```diff
@@ -29,6 +29,16 @@ public String home() {
     return "Hello World!";
   }
 
+  /**
+   * <a href="https://cloud.google.com/appengine/docs/flexible/java/how-instances-are-managed#health_checking">
+   * App Engine health checking</a> requires responding with 200 to {@code /_ah/health}.
+   */
+  @RequestMapping("/_ah/health")
+  public String healthy() {
+    // Message body required though ignored
+    return "Still surviving.";
```

Comment:
SGTM - I've been using Flex for two years a lot changes and I don't always see the updated docs.